### PR TITLE
Lease search and list updates psp-2224

### PR DESCRIPTION
**Pull request by FuriousLlama**

### CODE REVIEW

File: backend/.editorconfig [hunk 1]
```diff
@@ -76,7 +76,7 @@ dotnet_diagnostic.S4457.severity = error
 # S3442 "abstract" classes should not have "public" constructors
 dotnet_diagnostic.S3442.severity = error
 # S125 Remove commented out code
-dotnet_diagnostic.S125.severity = error
+dotnet_diagnostic.S125.severity = warning
```
Comment (devinleighsmith) [inline backend/.editorconfig:79]: Haha, nice.

Comment (devinleighsmith) [reply backend/.editorconfig:79]: I agree with this, it was kinda annoying during dev.

Comment (FuriousLlama) [reply backend/.editorconfig:79]: yeah, I meant to remove it earlier...

File: backend/dal/Services/LeaseService.cs [hunk 2]
```diff
@@ -139,7 +139,7 @@ public Lease Get(int id)
         /// <returns></returns>
         public Paged<Lease> GetPage(LeaseFilter filter)
         {
-            this.User.ThrowIfNotAuthorized(Permissions.PropertyView);
+            //This is not code this.User.ThrowIfNotAuthorized(Permissions.PropertyView);
```
Comment (devinleighsmith) [inline backend/dal/Services/LeaseService.cs:142]: this would be Permissions.ContactView

File: frontend/package.json [hunk 3]
```diff
@@ -18,6 +18,7 @@
     "lodash": "4.17.21",
     "moment": "2.29.1",
     "moment-timezone": "0.5.33",
+    "multiselect-react-dropdown": "^2.0.7",
```
Comment (devinleighsmith) [inline frontend/package.json:21]: remove the ^

**Review by devinleighsmith — APPROVED** [review]

**Pull request merged**